From f4a09e2edf4f00e97025922c15cf732407548a0a Mon Sep 17 00:00:00 2001
From: Bin Jiang <bin.jiang@windriver.com>
Date: Tue, 25 Nov 2014 14:36:58 +0800
Subject: [PATCH 6/7] realtek/r8168_n: fix a compile warning

drivers/net/ethernet/realtek/r8168_n.c:16908:6: warning: unused
	variable 'rx_quota' [-Wunused-variable]

Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 drivers/net/ethernet/realtek/r8168_n.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/realtek/r8168_n.c b/drivers/net/ethernet/realtek/r8168_n.c
index b8026e7..01e5e57 100644
--- a/drivers/net/ethernet/realtek/r8168_n.c
+++ b/drivers/net/ethernet/realtek/r8168_n.c
@@ -16905,7 +16905,6 @@ rtl8168_rx_interrupt(struct net_device *dev,
 {
 	unsigned int cur_rx, rx_left;
 	unsigned int delta, count = 0;
-	u32 rx_quota = RTL_RX_QUOTA(dev, budget);
 
 	assert(dev != NULL);
 	assert(tp != NULL);
@@ -16913,7 +16912,7 @@ rtl8168_rx_interrupt(struct net_device *dev,
 
 	cur_rx = tp->cur_rx;
 	rx_left = NUM_RX_DESC + tp->dirty_rx - cur_rx;
-	rx_left = rtl8168_rx_quota(rx_left, (u32) rx_quota);
+	rx_left = rtl8168_rx_quota(rx_left, (u32) RTL_RX_QUOTA(dev, budget));
 
 	if ((tp->RxDescArray == NULL) || (tp->Rx_skbuff == NULL))
 		goto rx_out;
-- 
1.7.5.4

